lib/services/nativescript-project-plugins-service.ts

```diff
@@ -0,0 +1,472 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import * as path from "path";
+import * as util from "util";
+import {EOL} from "os";
+import Future = require("fibers/future");
+import * as shelljs from "shelljs";
+import * as semver from "semver";
+
+import PluginsDataLib = require("../plugins-data");
+
+export class NativeScriptProjectPluginsService implements IPluginsService {
+	private static MESSAGES = ["NPM Packages", "NPM NativeScript Plugins", "Marketplace Plugins"];
```

Comment:
Another non-merge stopper, same as the previous occurrence of this variable - I'd rather it was named differently.